source/features/pinned-issues-update-time.tsx

```diff
@@ -42,7 +42,12 @@ async function init(): Promise<void | false> {
 	const lastUpdated: Record<string, IssueInfo> = await getLastUpdated(pinnedIssues.map(getPinnedIssueNumber));
 	for (const pinnedIssue of pinnedIssues) {
 		const issueNumber = getPinnedIssueNumber(pinnedIssue);
-		const {updatedAt} = lastUpdated[api.escapeKey(issueNumber)];
+		const {updatedAt} = lastUpdated[api.escapeKey(issueNumber)] || {};
+		// There was a pinned issue that was changed
```

Comment:
This should never happen. This is why I suggest that `cache.function` depends on the arguments received.

Suggested change instead of this one:

```ts
	cacheKey: ([issues]) => __filebasename + ':' + getRepoURL() + ':' + String(issues) // <- I think
```

Reply:
Worked